pkg/dev/debian9/Dockerfile

```diff
@@ -143,8 +143,8 @@ RUN apt-get install -y ruby ruby-dev rubygems gcc make \
 #use the following variables to choose the version of hubble
 ARG HUBBLE_CHECKOUT=develop
 ARG HUBBLE_GIT_URL=https://github.com/hubblestack/hubble.git
-ENV HUBBLE_VERSION=2.4.7_develop
-ENV HUBBLE_ITERATION=1
+ENV HUBBLE_VERSION=2.4.7
+ENV HUBBLE_ITERATION=50501
```

Comment:
I have prefixed the iteration number with "5050" in "pkg/dev" Dockerfiles ( since it looks like SOS; ideally one should not be running such builds on their prod envs )
I have assumed that iteration number will not go over 10.